src/lib/ts-parser.ts

```diff
@@ -122,26 +122,32 @@ export class TsParser {
             this.project = new Project({});
         }
         const fileGlobs: string[] = [];
-        fileGlobs.push(`'**/*.ts'`);
+        fileGlobs.push(`**/*.ts`);
         // Excluded added lasts because order determines what takes effect
         this.excludedDirs = makeExclusionsRelativeToSource(this.sourceDir, this.excludedDirs);
         this.excludedDirs.forEach((dir) => {
-            fileGlobs.push(`'!${dir}/**'`);
+            fileGlobs.push(`!${dir}/**`);
         });
         let rg_glob = '';
+        const rgGlobs = [];
         for (const fg of fileGlobs) {
             rg_glob += ` --glob ${fg}`;
+            rgGlobs.push('--glob');
```

Comment:
Small nit, this loops and array is redundant, you might as well update `fileGlobs` instead of creating yet another array.

Reply:
perhaps some debt items, didn't want to block the PR